media-video/gxine/gxine-0.5.909.ebuild

```diff
@@ -13,27 +13,32 @@ SLOT="0"
 KEYWORDS="amd64 ppc ppc64 x86"
 IUSE="l10n_cs l10n_de lirc nls nsplugin udev xinerama"
 
-COMMON_DEPEND=">=media-libs/xine-lib-1.1.20[gtk]
+DEPEND="
+	media-libs/xine-lib[gtk]
 	x11-libs/gtk+:2
-	>=dev-libs/glib-2
-	>=dev-lang/spidermonkey-1.8.2.15:0
+	dev-lang/spidermonkey:0
+	dev-libs/glib
 	x11-libs/libX11
 	x11-libs/libXext
 	x11-libs/libXrender
 	x11-libs/libxcb
 	lirc? ( app-misc/lirc )
 	nls? ( virtual/libintl )
-	nsplugin? ( dev-libs/nspr
+	nsplugin? (
+		dev-libs/nspr
 		x11-libs/libXaw
 		x11-libs/libXt )
 	udev? ( dev-libs/libgudev:= )
 	xinerama? ( x11-libs/libXinerama )"
-DEPEND="${COMMON_DEPEND}
+RDEPEND="${DEPEND}"
+BDEPEND="
 	virtual/pkgconfig
 	nls? ( sys-devel/gettext )"
-RDEPEND="${COMMON_DEPEND}"
+
+PATCHES=( "${FILESDIR}"/${P}-fno-common.patch )
```

Comment:
This is a nit but as a hint (and it can only ever be that) I do:
- ${P} for upstreamed patches;
- ${PN} for not.